src/geofiles/writer/geo_vrml_writer.py

```diff
@@ -112,8 +112,9 @@ def _write(
             vertex_mapping = dict()
```

Comment:
![image](https://user-images.githubusercontent.com/7925259/140048211-e95582be-3a96-447a-a886-64c735c61b5e.png)

Weird that errors wasn't there before, but somehow no occurs. Please check again.